### ISSUE TITLE
Want to create footer 

### ISSUE THREAD

**Issue by Lovegupta112** [purnima143/Pluto-UI#16]

**Comment by AdityaSingh-02** [comment]
Hey @Lovegupta112, Could you please provide description when you are Opening some issue

**Comment by Lovegupta112** [comment]
@AdityaSingh-02 i didn't get your question , could you please elaborate 🙂

**Comment by Lovegupta112** [comment]
Yes actually i want to make footer and inside footer there will be some links also some social media links etc.

**Comment by ManasMadan** [comment]
Please assign this issue to me @Lovegupta112, will create the design first share it with you then will start working on it

**Comment by Lovegupta112** [comment]
Hey actually i was waiting for  other website components but @ManasMadan bro , Do you want to make design of footer , am i right ??

**Comment by ManasMadan** [comment]
Yes, i want to work on the footer component

**Comment by Lovegupta112** [comment]
Actually i am also working on footer , is it possible to collaborate??

**Comment by ManasMadan** [comment]
@AdityaSingh-02 please assign the issue to me and @Lovegupta112

**Comment by AdityaSingh-02** [comment]
Hey @ManasMadan and @Lovegupta112 it would be great if you both can work together 💯

**Comment by Lovegupta112** [comment]
Yes sure @AdityaSingh-02

**Comment by ManasMadan** [comment]
Manas Madan | Contributor#6987

**Comment by ManasMadan** [comment]
Hi @AdityaSingh-02, not able to figure out how i can add footer to end of each story page. am able to add footer to end of each story. Can You Please help me with the same

**Comment by AdityaSingh-02** [comment]
Hey @ManasMadan , you are creating a new component right? eg. - inside src/components/footer? If you are doing like this, I don't think u need to add it to the footer of the storybook. Still, could you share screenshots of your work?

**Comment by ManasMadan** [comment]
@AdityaSingh-02 please assign someone else, am not able to do it

**Comment by vk20july** [comment]
@AdityaSingh-02 sir I want to work on the issue under gssoc'23

**Comment by AdityaSingh-02** [comment]
Hey @vk20july You've been already assigned with another issue please work on that first

**Comment by vk20july** [comment]
> Hey @vk20july You've been already assigned with another issue please work on that first

ok sir
